userspace/libscap/engine/gvisor/gvisor.cpp

```diff
@@ -189,7 +189,7 @@ static int32_t gvisor_get_stats(scap_engine_handle engine, scap_stats* stats)
 	return engine.m_handle->get_stats(stats);
 }
 
-static const struct scap_stats_v2* gvisor_get_stats_v2(scap_engine_handle engine, uint32_t flags, uint32_t* nstats, int32_t* rc)
+static const struct metrics_v2* gvisor_get_stats_v2(scap_engine_handle engine, uint32_t flags, uint32_t* nstats, int32_t* rc)
```

Comment:
Nit: I'd treat structs as first class types, which they are, in C++ and avoid prepending `struct`, or `class` for that matter as there is only a minor, conventional difference between the two.
This is probably the result of a search-and-replace and it just was already there. Sorry.